nav2_regulated_pure_pursuit_controller/test/test_regulated_pp.cpp

```diff
@@ -166,38 +160,6 @@ TEST(RegulatedPurePursuitTest, createCarrotMsg)
   EXPECT_EQ(rtn->point.z, 0.01);
 }
 
-TEST(RegulatedPurePursuitTest, findVelocitySignChange)
```

Comment:
make sure to add tests for your new methods (`isWithinInversionTolerances` `prunePlan` `removePosesAfterFirstInversion` `findFirstPathInversion`)